.github/workflows/test.yml

```
@@ -8,13 +8,14 @@ on:
 jobs:
 
   test:
-    name: ${{ matrix.platform }} ${{ matrix.python }}
+    name: ${{ matrix.platform }} ${{ matrix.python }} (${{ matrix.napari_version }})
     runs-on: ${{ matrix.platform }}
     strategy:
       fail-fast: false
       matrix:
         platform: [ubuntu-latest]
         python: [3.7, 3.8, 3.9]
+        napari_version: ["latest", "master"]
```

```suggestion
        napari_version: ["stable", "master"]
```
I think we'd used the word stable elsewhere for this (like in URLs), but maybe latest is fine too. Curious if @kne42 has a preference, I don't really mind